src/DynamoDbModel.php

```diff
@@ -195,10 +218,10 @@ public static function all($columns = [], $limit = -1)
         return $model->getAll($columns, $limit);
     }
 
-    public static function first($columns = [])
+    public function first($columns = [])
```

Comment:
There are issues around using a static::first() in a non-static context - which is common particularly when working with filtered results, e.g. Model::where($query)->first(). Using static::newInstance in this non-static context obliterates the state and the filter is lost. I'll look at how to accommodate both static and non-static use cases.